app/views/organisations/_show_organisation.html.erb

```diff
@@ -57,6 +57,8 @@
     <% end %>
   </div>
   <% if @organisation.ordered_corporate_information && @organisation.ordered_corporate_information.any? %>
-    <%= render partial: 'corporate_information' %>
+    <div class="organisation__corporate-information">
```

Comment:
What does this class do? It doesn't seem to be declared anywhere?

Reply:
Simply to help with the overriding problem mentioned above.

Reply:
In that case I'd suggest nesting it under the more general `.organisations` class?